app/serializers/v3/product_serializer.rb

```diff
@@ -11,7 +11,7 @@ def repositories
   end
 
   attributes :id, :name, :identifier, :former_identifier, :version, :release_type, :arch,
-             :friendly_name, :product_class, :cpe, :free, :description, :eula_url, :repositories, :product_type,
```

Comment:
Similar to the `upgrade_path_item_serializer.rb`, I'm not sure you can just remove this from the serializer.